lib/voyager-message/Cargo.toml

```diff
@@ -49,6 +49,7 @@ tracing                        = { workspace = true }
 tracing-subscriber             = { workspace = true, features = ["json"] }
 typenum                        = { workspace = true }
 unionlabs                      = { workspace = true, features = ["ethabi"] }
+voyager-core.workspace         = true
```

Comment:
I think we should go with `{ workspace = true }` everywhere

Reply:
yes i agree, however `cargo add` defaults to .workspace = true :(

Reply:
fixed all existing occurences of .workspace; ill look into formatting solutions for this

Reply:
https://github.com/unionlabs/union/issues/3007